### PR TITLE
Setup postgres

### PR DESCRIPTION
**Pull request by amosNash**
## How to Setup
npm run seedPostgres

## Description of Pull Request
• setup postgres database
• created summaries table
• created seeding script

### CODE REVIEW

File: database/postgresDB.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const { Sequelize } = require('sequelize');
+
+const db = new Sequelize({
+  database: 'summary',
+  username: 'root',
+  password: 'root',
```
Comment (christulin) [inline database/postgresDB.js]: I would make sure you move this username and password to a config file or env vars.. (I need to do the same)

File: database/seedPostgres.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+var randomIntLessThan = (input) => {
+  return Math.floor(Math.random() * input);
+}
+
+const PARTY_MODE = true;
+
+ const stayTypes = PARTY_MODE ?
+ ['Entire home', 'Private room', 'Treehouse', 'Entire bungalow', 'Entire camper', 'Studio apartment', 'Entire cabin', 'Private loft', 'Empty lot', 'Entire guest suite', 'Entire guesthouse', 'Entire condominium', 'Tiny house', 'Spy mansion', 'Haunted house', 'Bomb shelter', 'Bunkbed fort', 'Entire spacious trunk of car', 'Medieval castle', 'Entire Spice Bus', 'Airplane']
+ :
+ ['Entire home', 'Private room', 'Treehouse', 'Entire bungalow', 'Entire camper', 'Studio apartment', 'Entire cabin', 'Private loft', 'Empty lot', 'Entire guest suite', 'Entire guesthouse', 'Entire condominium', 'Tiny house'];
```
Comment (christulin) [inline database/seedPostgres.js]: Looks like `PARTY_MODE` is always true here, so wondering you you have this ternary operator to define `stayTypes`

**Review by christulin — APPROVED** [review]
Looks good to me

File: database/seedPostgres.js (hunk 2 above)
Comment (amosNash) [reply database/seedPostgres.js]: This was previously in the code base, but you are right that it can be taken out.

**Pull request merged**